pkg/config/shared.go

```diff
@@ -146,11 +146,19 @@ type RedisEvents struct {
 	} `name:"publish"`
 }
 
+// BatchEvents represents the configuration for batch event publication.
+type BatchEvents struct {
+	Enable bool          `name:"enable" description:"Enable events batching"`
+	Size   int           `name:"size" description:"How many items to have in a batch"`
+	Delay  time.Duration `name:"delay" description:"For how long to delay event submission in order to build a batch"`
+}
+
 // Events represents configuration for the events system.
 type Events struct {
 	Backend string      `name:"backend" description:"Backend to use for events (internal, redis, cloud)"`
 	Redis   RedisEvents `name:"redis"`
 	Cloud   CloudEvents `name:"cloud"`
+	Batch   BatchEvents `name:"batch"`
```

Comment:
Should this be marked as experimental?

Reply:
What is not in the changelog does not exist 😄 . But yes, we can mark these as experimental too.

In general I wish we had some kind of meta-issue with all of the experimental things, in order to ensure graduation at some point. We have one for feature flags, but not for things we mark as experimental in the API and configuration.

Reply:
Right. @KrishnaIyer can you create an issue template and file issues for experimental API and CLI commands?

That way at least we can track them.

I agree we should remove experimental after some time.